Orange/widgets/data/owfile.py

```diff
@@ -197,11 +213,37 @@ def browse_file(self, in_demos=False):
 
         self.add_path(filename)
         self.source = self.LOCAL_FILE
+        self.fill_sheet_combo(filename)
         self.load_data()
 
 
+    def fill_sheet_combo(self, path):
+        if os.path.exists(path) and self.is_multisheet_excel(path):
+            book = open_workbook(path)
+            sheets = book.nsheets
+            if sheets > 1:
+                self.sheet_label.setVisible(True)
+                self.sheet_combo.setVisible(True)
```

Comment:
It would be more elegant to put both controls into an hBox and hide the hBox. If you use `QGridLayout`, you will have to put them into such box anyway.